Makefile

```
@@ -48,3 +48,11 @@ compose-up:
 	docker-compose up
 
 compose-restart: compose-down codegen-compose compose-up
+
+# this is mentioned in nix/default.nix and should be removed at some point
+#install-kustomize:
+#	curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+#	export PATH=$PATH:$(realpath .)
+
+up-stage: install-kustomize
+	k8s/build.sh | oc apply -f -
```

We should drop the install-kustomize dependency, and we could inline the build command:

```suggestion
k8s-stage-up:
	kustomize build --load-restrictor LoadRestrictionsNone ./overlays/dev | oc apply -f -
```

---

PS, this would need to be changed to `k8s/overlays/dev`

---

done offline